tests/integration_tests/config/derive_photon_electron_spectrum_lst_ecsv.yml

```diff
@@ -9,14 +9,17 @@ CTA_SIMPIPE:
       AFTERPULSE_SPECTRUM: ./tests/resources/LST_afterpulses.ecsv
       STEP_SIZE: 0.02
       USE_NORM_SPE: true
+      FIT_AFTERPULSE: true
+      FIT_AFTERPULSE_DECAY_FACTOR: 15
+      AFTERPULSE_AMPLITUDE_RANGE: [1, 42]
```

Comment:
Range is different here than the default and the test below.

Reply:
Yes - an integration test can/should(?) use values different from default.

Reply:
OK!